src/main/scala/com/databricks/labs/overwatch/pipeline/BronzeTransforms.scala

```diff
@@ -349,8 +350,18 @@ trait BronzeTransforms extends SparkSessionWrapper {
         .withColumn("timestamp", unix_timestamp('time) * 1000)
         .withColumn("date", 'time.cast("date"))
         .select('category, 'version, 'timestamp, 'date, 'properties, 'identity.alias("userIdentity"), 'organization_id)
-        .selectExpr("*", "properties.*").drop("properties")
-
+        .selectExpr(
+          "*",
+          "properties.actionName",
+          "properties.logId",
+          "properties.requestId",
+          "properties.requestParams",
+          "properties.response",
+          "properties.serviceName",
+          "properties.sessionId",
+          "properties.sourceIPAddress",
+          "properties.userAgent")
+        .drop("properties")
```

Comment:
Can we add these column list in Schema.scala and then use this list instead of passing the hardcoded column names ?